components/style/properties/longhands/font.mako.rs

```diff
@@ -97,7 +93,6 @@ ${helpers.predefined_type(
     "font-stretch",
     "FontStretch",
     engines="gecko servo-2013 servo-2020",
-    servo_2020_pref="layout.2020.unimplemented",
```

Comment:
Do we have layout code for this one? (Or in fact for all properties enabled here?)

Reply:
We pass that in the font style struct, so kinda, but not really.